client/src/utils/requestHandlers.js

```diff
@@ -0,0 +1,39 @@
+// Import Axios
+const axios = require("axios");
+
+// Import reducers
+// #NOTE I do not fully understand how to update the reducer. I know what is does, but not how to code its usage
+
+// Handle all requests to API here
+module.exports = {
+  async getFromAPI(config) {
+    try {
+      // Logic to update reducer (Sent)
+      const response = await axios(config);
+
+      //Logic to update reducer (Success)
+      return response;
```

Comment:
This would be (eg) 
```js
dispatch({ type: 'FETCH_LANGUAGES_SUCCESS, response });
```

Reply:
Having issues with dispatch. Dispatch is not a function when I try and call it. I've passed it correctly, but nothing seems to be working there. The code is still functional without it, so it is clearly just dispatch,